tools/azure-sdk-tools/devtools_testutils/helpers.py

```diff
@@ -86,6 +88,22 @@ def is_preparer_func(fn):
     return getattr(fn, "__is_preparer", False)
 
 
+def create_random_name(prefix="aztest", length=24):
+    if len(prefix) > length:
+        raise ValueError("The length of the prefix must not be longer than random name length")
+
+    padding_size = length - len(prefix)
+    if padding_size < 4:
+        raise ValueError(
+            "The randomized part of the name is shorter than 4, which may not be able to offer enough randomness"
+        )
+
+    random_bytes = os.urandom(int(math.ceil(float(padding_size) / 8) * 5))
+    random_padding = base64.b32encode(random_bytes)[:padding_size]
```

Comment:
Depending on how complex you want to make this. Compare with simpler code:

```python
import random
import string
''.join(random.choice(string.ascii_lowercase) for _ in range(25))
```

![image](https://github.com/Azure/azure-sdk-for-python/assets/45376673/10483b97-1543-4b33-8d6b-3c1496740800)